stream_alert_cli/runner.py

```diff
@@ -41,8 +38,7 @@ def cli_runner(options):
             Contains the following keys for lambda commands:
                 (command, subcommand, env, func, source)
     """
-    cli_load_message = ('Issues? Report here: '
-                        'https://github.com/airbnb/streamalert/issues')
+    cli_load_message = ('Issues? Report here: https://github.com/airbnb/streamalert/issues')
```

Comment:
doesn't pylint complain about unnecessary parenthesis here?

Reply:
apparently not, i'll fix though